Dockerfile

```diff
@@ -134,7 +134,7 @@ RUN \
 ADD config /etc/nginx/
 ADD *.sh /usr/bin/
 
-VOLUME /etc/nginx/conf.d /var/log/nginx
+VOLUME /etc/nginx/conf.d /etc/nginx/lua /var/log/nginx
```

Comment:
Then why /etc/nginx/canary and not /etc/nginx/latest... I think I'm missing something.